### PR TITLE
Test against latest ruby and rails

### PR DESCRIPTION
**Pull request by tagliala**
Rails 6.0 is Round the Corner

This PR checks that everything is fine with RoR 6.0.0's first release candidate

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,34 +19,55 @@ gemfile:
...
     - rvm: ruby-head
+    - rvm: 2.4
+      gemfile: gemfiles/rails_3.gemfile
     - rvm: 2.4
       gemfile: gemfiles/rails_4.gemfile
     - rvm: 2.4
       gemfile: gemfiles/rails_4.1.gemfile
-    - rvm: 2.4
-      gemfile: gemfiles/rails_4.2.gemfile
+    - rvm: 2.5
```
Comment (januszm) [inline .travis.yml]: We don't need a test for Rails 4.2 on Ruby 2.4 ? Why it was removed?

Comment (tagliala) [reply .travis.yml]: it has been removed from `allow_failures:`: this test is not supposed to fail

image

Comment (tagliala) [reply .travis.yml]: ref: commit message b77d9d3a9bbbeaaf005364acae209439f2eeff61

Comment (januszm) [reply .travis.yml]: Oh that's `allow_failures`, ✅

**Comment by januszm** [comment]
Thank you @tagliala !

**Pull request merged**